### PR TITLE
Add Resource List to Ansible User Guide

### PR DESCRIPTION
**Pull request by bradtopol**
Add Resource List to Ansible User Guide

Closes: #2072

### CODE REVIEW

File: doc/ansible/user-guide.md [hunk 1]
```diff
@@ -441,6 +441,13 @@ $ kubectl delete -f deploy/crds/cache.example.com_memcacheds_crd.yaml
...
 **NOTE** Additional CR/CRD's can be added to the project by running, for example, the command :`operator-sdk new api --api-version=cache.example.com/v1alpha1 --kind=AppService --type=ansible`
 
+## Ansible Resources
+
+- [Ansible Testing Guide][ansible-test]
+- [Ansible Developer Guide][ansible-developer]
+
+[ansible-test]:./dev/testing_guide.md
+[ansible-developer]:./dev/developer_guide.md
 [operator-scope]:./../operator-scope.md
```
Comment (camilamacedo86) [inline doc/ansible/user-guide.md:451]: We are migrating to haven a website to host all docs. 
This process in ongoing. See: https://deploy-preview-2841--operator-sdk.netlify.com/

Screenshot 2020-04-15 at 02 14 14

So, any change in the docs needs to be replicated to the website dir/structure in the docs.
Note that you might need to change the links to make it work there.

Comment (bradtopol) [reply doc/ansible/user-guide.md:451]: @camilamacedo86 So I did not see a copy of the file in the corresponding place in the en folder. Should I just add a copy of the file in the corresponding place?

Comment (camilamacedo86) [reply doc/ansible/user-guide.md:451]: Hi @bradtopol, 

First at all, I do not think that this change matches with the new "layout of the docs". See that for Ansible has more docs/resources than the list added here. Also, they will be available on the menu. So, I particularly do not think that we should move forward with.  

Regards your question you need to find out the correspondent doc there. in this case it is ; https://github.com/operator-framework/operator-sdk/blob/master/website/content/en/docs/ansible/quickstart.md

However, IHMO the changes made here would not fit in the "new" docs format so in POV we should not move forward with. Is it make sense?

Comment (bradtopol) [reply doc/ansible/user-guide.md:451]: Hi @camilamacedo86 That makes sense to me.   Do you want to just close my PR and also close the issue saying its been addressed by the new layout of the docs?

Comment (camilamacedo86) [reply doc/ansible/user-guide.md:451]: It sounds great.
Really thank you for your contribution.

**Comment by camilamacedo86** [comment]
/close 

As agreed in the comment https://github.com/operator-framework/operator-sdk/pull/2841#discussion_r409010712 it is already something that will be solved with the website docs.

**Pull request closed without merging**